composer.json

```diff
@@ -1,32 +1,75 @@
 {
-  "name": "cwp/cwp-installer",
-  "description": "CWP Project Template",
-  "license": "BSD-3-Clause",
-  "require": {
-    "cwp/cwp-recipe-basic": "1.8.x-dev",
-    "cwp/cwp-recipe-blog": "1.8.x-dev",
-    "cwp/starter-theme": "1.1.x-dev"
-  },
-  "require-dev": {
-    "cwp/cwp-recipe-basic-dev": "^1.1@stable"
-  },
-  "suggest": {
-    "silverstripe/textextraction": "Required for document text searching within CWP"
-  },
-  "config": {
-    "process-timeout": 900
-  },
-  "extra": {
-    "branch-alias": {
-      "dev-master": "1.8.x-dev"
-    }
-  },
-  "repositories": [
-    {
-      "type": "composer",
-      "url": "https://packages.cwp.govt.nz/"
-    }
-  ],
-  "prefer-stable": true,
-  "minimum-stability": "dev"
+    "name": "cwp/cwp-installer",
+    "type": "silverstripe-recipe",
+    "description": "CWP Project Template",
+    "license": "BSD-3-Clause",
+    "require": {
+        "php": ">=5.6.0",
+        "silverstripe/recipe-plugin": "^1",
+        "silverstripe/recipe-core": "1.1.x-dev",
+        "silverstripe/assets": "1.1.x-dev",
+        "silverstripe/config": "1.0.x-dev",
+        "silverstripe/framework": "4.1.x-dev",
+        "silverstripe/recipe-cms": "1.1.x-dev",
+        "silverstripe/admin": "1.1.x-dev",
+        "silverstripe/asset-admin": "1.1.x-dev",
+        "silverstripe/campaign-admin": "1.1.x-dev",
+        "silverstripe/cms": "4.1.x-dev",
+        "silverstripe/errorpage": "1.1.x-dev",
+        "silverstripe/graphql": "1.1.x-dev",
+        "silverstripe/reports": "4.1.x-dev",
+        "silverstripe/siteconfig": "4.1.x-dev",
+        "silverstripe/versioned": "1.1.x-dev",
+        "cwp/cwp-recipe-core": "2.x-dev",
+        "cwp/cwp-core": "2.x-dev",
+        "silverstripe/hybridsessions": "2.0.x-dev",
+        "silverstripe/auditor": "2.x-dev",
+        "silverstripe/environmentcheck": "2.x-dev",
+        "cwp/cwp": "2.x-dev",
+        "silverstripe/html5": "2.x-dev",
+        "symbiote/silverstripe-gridfieldextensions": "3.0.x-dev",
+        "cwp/cwp-pdfexport": "1.x-dev",
+        "cwp/cwp-search": "1.x-dev",
+        "symbiote/silverstripe-queuedjobs": "4.x-dev",
+        "silverstripe/subsites": "2.0.x-dev",
+        "tractorcow/silverstripe-fluent": "4.0.x-dev",
+        "silverstripe/registry": "2.x-dev",
+        "cwp/starter-theme": "2.x-dev"
+    },
+    "suggest": {
+        "silverstripe/textextraction": "3.x-dev requires framework 4.0.x, but CWP needs 4.1.x (part of cwp/recipe-search)"
```

Comment:
Do we need to make a PR to change the constraint then?

Reply:
So long as we test it too, sure.

Reply:
I've changed it to ^4 in this pull request which will let us use it on both: https://github.com/silverstripe/silverstripe-textextraction/pull/41

Caveat: that module is a WIP at the moment

Reply:
Ah, that'll be why I saw the tests were failing.
I guess failing tests don't affect composer, so I'll upgrade this to a requirement in the cwp-search segment.